src/shared/filesystemUtilities.ts

```diff
@@ -12,8 +12,10 @@ import * as path from 'path'
 import * as vscode from 'vscode'
 import { getLogger } from './logger'
 import * as pathutils from './utilities/pathUtils'
+import globals from '../shared/extensionGlobals'
 
 const DEFAULT_ENCODING: BufferEncoding = 'utf8'
+export const latestDownloadPathSetting = 'aws.latestDowloadPath'
```

Comment:
not a setting, just a key.

Don't need this constant anyway, instead make a set function similar to the get function (getDefaultDownloadPath)

Reply:
"recent" seems to be the preferred word for these things although `aws.downloadPath` (or `aws.downloadLocation`) would be good too. The adjective feels unnecessary IMO.